benchmark/fs/read-stream-throughput.js

```diff
@@ -5,19 +5,20 @@ const path = require('path');
 const common = require('../common.js');
 const filename = path.resolve(__dirname, '.removeme-benchmark-garbage');
 const fs = require('fs');
-const filesize = 1000 * 1024 * 1024;
 const assert = require('assert');
 
-var type, encoding, size;
+let type, encoding, size, filesize;
 
 const bench = common.createBenchmark(main, {
   type: ['buf', 'asc', 'utf'],
+  filesize: [1000 * 1024 * 1024],
   size: [1024, 4096, 65535, 1024 * 1024]
 });
 
 function main(conf) {
   type = conf.type;
   size = +conf.size;
+  filesize = conf.filesize
```

Comment:
Linter shows this line is missing a semicolon.
```
not ok 5 - /usr/home/iojs/build/workspace/node-test-linter/benchmark/fs/read-stream-throughput.js
  ---
  message: Missing semicolon.
  severity: error
  data:
    line: 21
    column: 27
    ruleId: semi
  ...
```

Reply:
Sorry, just pushed the fix